### PR TITLE
Removal of role of executive-members and some points on offboarding executives.

### PR DESCRIPTION
**Pull request by rava-dosa**
Change in path in :-
- `/community/governance`

### CODE REVIEW

**Comment by rava-dosa** [comment]
ping @americast @djbarnwal @sangeet259 @dibyadas

File: community/governance.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ toc=true
   * **Advisors**: People who have been an Executive member in the past and are still on the campus.
   * **Alumni**: People who have graduated but were Executives when on campus.
 
-Permanent membership of KOSS mandates serving one year as an Executive (either as Head or Member).
+Permanent membership of KOSS mandates serving one year as an Executive. But if advisors mutually agree that any particular human shouldn't be advisor due to several reasons like fundamentals, or their contribution as Executive then they can remove that particular human.
```
Comment (dibyadas) [inline community/governance.md]: This is good. I wonder if we can elaborate the word 'fundamentals' here to avoid any ambiguity.

Comment (rava-dosa) [reply community/governance.md]: Hi @dibyadas pushed new commit, please checkout, If I haven't introduced new ambiguity in order to resolve the old one.

**Comment by rava-dosa** [comment]
This pr has 7 approvals, hence merging this. Thanks for taking time out of your busy schedule and reviewing this.

**Pull request merged**